packages/grid/_modules_/grid/components/cell/GridCell.tsx

```diff
@@ -145,13 +145,17 @@ export const GridCell = React.memo(function GridCell(props: GridCellProps) {
   };
 
   React.useLayoutEffect(() => {
-    const doc = ownerDocument(apiRef!.current.rootElementRef!.current as HTMLElement);
+    if (!hasFocus) {
+      return;
+    }
+
+    if (cellMode === 'edit' && isEditable) {
+      return;
+    }
+
+    const doc = ownerDocument(apiRef!.current.rootElementRef!.current as HTMLElement)!;
 
-    if (
-      hasFocus &&
-      cellRef.current &&
-      (!doc.activeElement || !cellRef.current!.contains(doc.activeElement))
-    ) {
+    if (cellRef.current && (doc.activeElement || !cellRef.current!.contains(doc.activeElement))) {
```

Comment:
AFAIK we will never see doc.activeElement empty, unless on IE https://github.com/sveltejs/sapper/issues/332#issue-346813546

```suggestion
    if (cellRef.current && !cellRef.current!.contains(doc.activeElement!)) {
```